docs/content/en/blog/news/february-2022-update.md

```diff
@@ -0,0 +1,97 @@
+---
+date: 2022-02-10
+title: "February 2022 update"
+linkTitle: "February 2022 update"
+weight: 996
+description: "Development status update to recap what happened in January"
+author: Le Van Nghia ([@nghialv](https://twitter.com/nghialv2607))
+---
+
+_Published by the PipeCD dev team every month, this update will provide you with news and updates about the project! Please click [here](/blog/2022/01/05/january-2022-update/) if you want to see the last status update._
+
+### What's changed
+---
+
+Since the last report, PipeCD team has introduced 4 releases ([v0.24.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.0), [v0.24.5](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.5), [v0.25.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.25.0), [v0.25.1](https://github.com/pipe-cd/pipecd/releases/tag/v0.25.1)). This blog post walks you through their notable changes. For all other changes, please check out each release note to see.
```

Comment:
```suggestion
Since the last report, PipeCD team has introduced 4 releases ([v0.24.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.0), [v0.24.5](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.5), [v0.25.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.25.0), [v0.25.1](https://github.com/pipe-cd/pipecd/releases/tag/v0.25.1)). This blog post walks you through their notable changes. For all other changes, please check out each release note.
```
nits